client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -172,6 +172,7 @@ class SiteImporterInputPane extends Component {
 			},
 			site: this.props.site,
 			targetSiteUrl: urlForImport,
+			targetPlatform: 'wix',
```

Comment:
I would avoid hardcoding target platform from the reusable `SiteImporterInputPane` component. Instead, you can provide the dynamic `engine` property from the parent component (index.jsx).

Reply:
Indeed, we should not hardcode the target platform in a reusable component. Hardcoding target platform removed https://github.com/Automattic/wp-calypso/pull/64881/commits/1c923d3c83f1d491611c2504d16df61c9f0db236

Reply:
We don't need to manually specify the platform. There is a property named `engine`, so can simply consume it.
The property path is: `this.props.importerData.engine` and the value is a selected engine, and for this particular case the value is `wix`.

Reply:
Good to know! Fixed at https://github.com/Automattic/wp-calypso/pull/64881/commits/143c1f40613ddf5ade08f7971335551f5e046b08

Reply:
👍